ui/src/gear/hood/types.ts

```diff
@@ -161,20 +162,15 @@ export interface Vat {
 export interface Vats {
   [desk: string]: Vat;
 }
+
 /**
  * TODO: crisp one-liner describing a Pike
```

Comment:
The People clamor to know: What is a Pike?

Reply:
"Update config & status for desk" or something along those lines?